Extension/i18n/csy/walkthrough/installcompiler/install-compiler-windows.md.i18n.json

```diff
@@ -17,9 +17,9 @@
 	"walkthrough.windows.note1": "Poznámka",
 	"walkthrough.windows.note1.text": "Můžete použít sadu nástrojů C++ z Visual Studio Build Tools spolu s Visual Studio Code ke kompilaci, sestavení a ověření jakékoli kódové báze C++, pokud máte také platnou licenci Visual Studio (buď Community, Pro nebo Enterprise), kterou aktivně používáte k vývoji kódové báze C++.",
 	"walkthrough.windows.open.command.prompt": "Otevřete {0} zadáním příkazu „developer“ do nabídky Start systému Windows.",
-	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt for VS",
+	"walkthrough.windows.command.prompt.name1": "Developer Command Prompt pro VS",
```

Comment:
@bobbrow Aren't these translations incorrect? i.e. shouldn't they be fixed manually?

Reply:
Yes. I planned to follow up on that one separately, since we translate some and not others in this language.  It's about half `for` and half `pro` right now. I was just trying to get some loc update in this pre-release so I didn't include that as part of this PR.

Reply:
@bobbrow I don't understand. Can't we just fix it ourselves by forcing them all to be `Developer Command Prompt for VS` until we get the loc team to fix it?

Reply:
i.e. it appears that's what we had done previously and this PR is now reverting that fix?

Reply:
I can fix this one if it bothers you, but I'm not going to do a pass on all of them right now because it will increase the noise on these Loc PR's until I get around to opening all the bugs.

Reply:
@bobbrow Yeah, can you fix the `Developer Command Prompt for VS` cases? That's what we normally do. Unless Colen thinks it's okay.

Reply:
I'm just fixing the ones you pointed out. These add noise to the branch until we get the bugs opened though.